src/aui-form-validator/tests/unit/js/tests.js

```diff
@@ -190,6 +190,48 @@ YUI.add('aui-form-validator-tests', function(Y) {
             Y.Assert.isFalse(validator.hasErrors());
         },
 
+        /*
+         * Check if validator correctly validates dates in pt-br format dd/MM/yyyy
+         * @tests AUI-1940
+         */
+          'test date format': function() {
```

Comment:
@linolaoj  This test are falling on Mozilla Firefox. Could you please check this?